python/cuxfilter/charts/datashader/datashader.py

```diff
@@ -225,13 +226,162 @@ def scatter(
+    node_aggregate_fn="count",
+    edge_aggregate_fn="count",
+    node_color_palette=CUXF_DEFAULT_COLOR_PALETTE,
+    edge_color_palette=["#000000"],
+    node_point_size=8,
+    node_point_shape="circle",
+    node_pixel_shade_type="eq_hist",
+    node_pixel_density=0.8,
+    node_pixel_spread="dynspread",
+    tile_provider=None,
+    width=800,
+    height=400,
+    title="",
+    timeout=1,
+    **library_specific_params,
+):
+
+    """
+    Parameters
+    ----------
+    node_x: str, default "x"
+        x-coordinate column name for the nodes cuDF dataframe
+
+    node_y: str, default "y"
+        y-coordinate column name for the nodes cuDF dataframe
+
+    node_id: str, default "vertex"
+        node_id/label column name for the nodes cuDF dataframe
+
+    edge_source: str, default "source"
+        edge_source column name for the edges cuDF dataframe
+
+    edge_target="target",
+        edge_target column name for the edges cuDF dataframe
+
+    x_range: tuple, default(nodes_gpu_dataframe[x].min(),
+        nodes_gpu_dataframe[x].max())
+        (min, max) x-dimensions of the geo-scatter plot to be displayed
+
+    y_range: tuple, default(nodes_gpu_dataframe[y].min(),
+    nodes_gpu_dataframe[y].max())
+        (min, max) x-dimensions of the geo-scatter plot to be displayed
+
+    add_interaction: {True, False},  default True
+
+    node_aggregate_col=str, default None,
+        column from the nodes gpu dataframe on which the mode_aggregate_fn
+        will be run on
+
+    edge_aggregate_col=str, default None,
+        column from the edges gpu dataframe on which the mode_aggregate_fn
+        will be run on
+
+    node_aggregate_fn={'count', 'mean', 'max', 'min'},  default 'count'
+    edge_aggregate_fn={'count', 'mean', 'max', 'min'},  default 'count'
+
+    node_color_palette=bokeh.palettes or list of hex_color_codes, or list of
+                   color names,  default CUXF_DEFAULT_COLOR_PALETTE(viridis10)
+
+    edge_color_palette=bokeh.palettes or list of hex_color_codes, or list of
+                   color names,  default ["#000000"]
+
+    node_point_size: int, default 8
+        Point size in the scatter plot.
+
+    node_point_shape: str, default 'circle'
+        Available options: circle, square, rect_vertical, rect_horizontal.
+
+    node_pixel_shade_type: str, default 'eq_hist'
+        The "how" parameter in datashader.transfer_functions.shade()
+        function.
+        Available options: eq_hist, linear, log, cbrt
+
+    node_pixel_density: float, default 0.8
+        A tuning parameter in [0, 1], with higher values giving more dense
+        scatter plot.
+
+    node_pixel_spread: str, default 'dynspread'
+        dynspread: Spread pixels in an image dynamically based on the image
+        density.
+        spread: Spread pixels in an image.
+
+    tile_provider: str, default None
+        Underlying map type.See
+        https://bokeh.pydata.org/en/latest/docs/reference/tile_providers.html
+
```

Comment:
this link is old, should be

https://docs.bokeh.org/en/latest/docs/reference/tile_providers.html

Reply:
It looks like the old domain is in the README too

Reply:
fixed